app/story/story.py

```diff
@@ -31,6 +33,17 @@ def make_analyst_appointment_validator(user_id):
     # this is only on app side, we don't need to validate things here
     return True
 
+# server actions
+def telegram_highjack(user_id):
```

Comment:
Can not test....

Reply:
try again with the app or send phonenumber manually first, then activate the `end` story_point, wait for your login code and send it manually to the server - then it should work